### DISCUSSION TITLE
JDK version support policy

### DISCUSSION THREAD

**Discussion by zhouyuan** [apache/gluten#8272] [Polls]
Spark dropped JDK8 support in 4.0, as well as in recent version of Hadoop/Flink. However these projects are doing branch based development, which means the old branch will not greatly get impacted. 
Gluten is designed to have support for old and new Spark release with same code. Thus the supporting efforts is different than vanilla Spark. 
We are trying to collect more info for JDK version used in production env so that we could align better on the coming upgrade. Please vote in below poll. 

Thanks, -yuan

**Comment by FelixYBW** [comment]
can we set if customer build for spark4.0, they must use JDK17?
